docs/documentation/advanced/term/term.mdx

```diff
@@ -81,3 +85,7 @@ WHERE id @@@
 }'::jsonb;
 ```
 </CodeGroup>
+
+## Notes
+
+Avoid using term query for text field as term query does not analyze the search term. The term query looks for the exact term you provide and hence this may return poor or no results.
```

Comment:
I don't know if this is necessary

Reply:
Agreed, this is not really an accurate portrayal of term queries.

Reply:
yes i have removed this. I think line we have added in basic usage will suffice.